.eslintrc.json

```diff
@@ -1,47 +1,26 @@
 {
-  "extends": "@appium/eslint-config-appium",
+  "extends": ["plugin:react/recommended", "@appium/eslint-config-appium"],
```

Comment:
Thanks for doing this. How about switching from babel to ts and doing the corresponding change for the linter as well? Similarly to what was done in https://github.com/appium/appium-mac2-driver/pull/216

typescript compiler does not require everything to be rewritten into TS, but it can also validate vanilla JS files and does it good.

Reply:
Thanks, I'll take a look at this in a separate task. There will probably be some differences because babel isn't directly used to build the app, so I'm not sure how that will change.